<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[database connected]]>
    </line>
    <line>
      <![CDATA[test staeted]]>
    </line>
    <line>
      <![CDATA[databse disconnected]]>
    </line>
  </reporter-output>
  <suite started-at="2022-12-05T13:08:45 IST" name="Default suite" finished-at="2022-12-05T13:09:14 IST" duration-ms="28565">
    <groups>
    </groups>
    <test started-at="2022-12-05T13:08:45 IST" name="Default test" finished-at="2022-12-05T13:09:14 IST" duration-ms="28565">
      <class name="com.Actitime.Testscripts.CreateCustomer">
        <test-method is-config="true" signature="databaseconnection()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:45 IST" name="databaseconnection" finished-at="2022-12-05T13:08:45 IST" duration-ms="10" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[database connected]]>
            </line>
          </reporter-output>
        </test-method> <!-- databaseconnection -->
        <test-method is-config="true" signature="launchbrowser()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:45 IST" name="launchbrowser" finished-at="2022-12-05T13:08:47 IST" duration-ms="2298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchbrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:47 IST" name="login" finished-at="2022-12-05T13:08:53 IST" duration-ms="5617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="creatcustomer()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:53 IST" name="creatcustomer" finished-at="2022-12-05T13:08:53 IST" duration-ms="15" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Syntax error, insert "Identifier (" to complete MethodHeaderName
Syntax error, insert ")" to complete MethodDeclaration
Syntax error, insert ";" to complete MethodDeclaration
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error, insert "Identifier (" to complete MethodHeaderName
Syntax error, insert ")" to complete MethodDeclaration
Syntax error, insert ";" to complete MethodDeclaration

at com.Actitime.pom.HomePage.<init>(HomePage.java:49)
at com.Actitime.Testscripts.CreateCustomer.creatcustomer(CreateCustomer.java:19)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
            <line>
              <![CDATA[test staeted]]>
            </line>
          </reporter-output>
        </test-method> <!-- creatcustomer -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:53 IST" name="logout" finished-at="2022-12-05T13:08:57 IST" duration-ms="4018" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Syntax error, insert "Identifier (" to complete MethodHeaderName
Syntax error, insert ")" to complete MethodDeclaration
Syntax error, insert ";" to complete MethodDeclaration
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error, insert "Identifier (" to complete MethodHeaderName
Syntax error, insert ")" to complete MethodDeclaration
Syntax error, insert ";" to complete MethodDeclaration

at com.Actitime.pom.HomePage.<init>(HomePage.java:49)
at com.Actitime.Generic.Baseclass.logout(Baseclass.java:65)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:08:57 IST" name="closebrowser" finished-at="2022-12-05T13:09:14 IST" duration-ms="16255" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
        <test-method is-config="true" signature="databasedisconnected()[pri:0, instance:com.Actitime.Testscripts.CreateCustomer@3835c46]" started-at="2022-12-05T13:09:14 IST" name="databasedisconnected" finished-at="2022-12-05T13:09:14 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[databse disconnected]]>
            </line>
          </reporter-output>
        </test-method> <!-- databasedisconnected -->
      </class> <!-- com.Actitime.Testscripts.CreateCustomer -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
